stdlib/public/core/TemporaryAllocation.swift

```diff
@@ -88,13 +88,18 @@ internal func _isStackAllocationSafe(byteCount: Int, alignment: Int) -> Bool {
     return true
   }
 
+#if !$Embedded
   // Finally, take a slow path through the standard library to see if the
   // current environment can accept a larger stack allocation.
   guard #available(macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4, *) //SwiftStdlib 5.6
   else {
     return false
   }
   return swift_stdlib_isStackAllocationSafe(byteCount, alignment)
+#else
+  return false
```

Comment:
shouldn't this be always true? 'cause else wise it forces the temp allocations to be heap based

Reply:
See a few lines above -- there's a hardcoded limit of 1024, under which stack allocations are always allowed. Returning false here means allocations over 1024 move to the heap. That seems pretty reasonable to me, though of course 1024 is a very arbitrary limit...

Reply:
well in a number of embedded systems the heap and stack are from the same ram segment so eh 6 one way 1/2 a dozen the other I guess...